### ISSUE TITLE
Launcher not launching

### ISSUE THREAD

**Issue by Artcp** [remyroy/CDDA-Game-Launcher#472]
I downloaded the latest version of the cdda launcher (1.4.11) and it doesn't launch. I installed it alright and it just doesn't open when I try to.
I used the 1.3 version before with no problems and I could update the game, but I can't seem to update the game with this version anymore, so I have to use the 1.4 version.

**Desktop (please complete the following information):**
 - OS: Windows 10 x64

**Additional context**
I have downloaded the Microsoft Visual C++ Redistributable as linked in https://github.com/remyroy/CDDA-Game-Launcher/releases/tag/v1.4.11
I have turned my firewall off.
I have turned my antivirus off.
I have executed as administrator.
Nothing worked.

**Comment by remyroy** [comment]
I've just built a debug release. You can find it here: https://www.dropbox.com/s/pubcy0putxbmsde/cddagl_portable_v1.4.11.debug.py37.zip?dl=1

Download this release, start the launcher with `launcher-debug.bat`. It will create a file called *debug.txt*. Send that file back to me. It might provide more information regarding your issue to help me fix it.

**Comment by Artcp** [comment]
It did launch this way. Here is the debug file.
[debug.txt](https://github.com/remyroy/CDDA-Game-Launcher/files/4879723/debug.txt)

I was able to update the game. :)

**Comment by remyroy** [comment]
I'm glad that worked for you. This debug version should be also the same as the 1.4.11 release version so I'm somewhat confused as to what is wrong with your system and that version. If that works for you, keep using this version.

**Comment by chronowarden** [comment]
Hello, I'd just like to say that I had the same problem as this other user and the debug version also fixed it for me.

**Comment by remyroy** [comment]
@chronowarden Hmm, that is kinda weird. Maybe there is an issue with the release build. I've just rebuilt the release version. Can you try it and tell me if it works for you. This is a normal release without the debug info: https://www.dropbox.com/s/olnr3dr4arh1jgr/cddagl_portable_v1.4.11.rebuilt.zip?dl=1

**Comment by chronowarden** [comment]
Yeah, that one is also not starting for me. Only the debug and some older versions of the launcher seems to work.

**Comment by Artcp** [comment]
Sorry for taking so long to answer, my problem was solved and I've been happily playing. Couldn't launch the normal release either.
